platform/viewer/src/index.js

```diff
@@ -13,19 +13,11 @@ import ReactDOM from 'react-dom';
  * EXTENSIONS
  * =================
  *
- * Importing and modifying the extensions our app uses HERE allows us to leverage
- * tree shaking and a few other niceties. However, by including them here they become
- * "baked in" to the published application.
- *
- * Depending on your use case/needs, you may want to consider not adding any extensions
+ * You may want to consider not adding any extensions
```

Comment:
haha

Reply:
Hahaha... I knew deep down that I would go back, I just wanna check with you how ill add those inside the config function before we call it inside app to inject the deps. This should go as a separate prop?